package.json

```
@@ -43,21 +46,23 @@
   },
   "devDependencies": {
     "@edx/browserslist-config": "^1.1.1",
-    "@edx/frontend-platform": "7.0.1",
-    "@openedx/frontend-build": "13.0.27",
+    "@edx/frontend-platform": "8.0.0",
+    "@openedx/frontend-build": "14.0.3",
```

Is there a specific reason to pin these at exact versions instead of `^` versions?

```suggestion
    "@edx/frontend-platform": "^8.0.0",
    "@openedx/frontend-build": "^14.0.3",
```

---

No specific reason, I have made the required update